CHANGES/790.misc.rst

```diff
@@ -0,0 +1 @@
+
```

Comment:
An empty change note is probably not the best idea. Either fill it out or I'll apply a label for the bot to just ignore it. I'm actually not sure what Towncrier would do with an empty fragment...

Reply:
I can certainly remove it, but it is a normal option with towncrier.  That way it's all 'in the system' rather than you having to add external tooling to sometimes ignore the check via a GitHub PR label as it sounds like you have.  I'd link to the twisted contributing docs but...  the link I found is 404.

You don't appear to be configuring custom sections in the changelog.

https://github.com/aio-libs/yarl/blob/68e7d9312ad9eccb695dbc174a4705e9a238c1fa/pyproject.toml
```toml
[tool.towncrier]
package = "yarl"
filename = "CHANGES.rst"
directory = "CHANGES/"
title_format = "{version} ({project_date})"
issue_format = "`#{issue} <https://github.com/aio-libs/yarl/issues/{issue}>`_"
```

So you would be using the defaults where, as I recall, `misc` has `showcontent` set to false.

https://towncrier.readthedocs.io/en/stable/configuration.html?highlight=showcontent#custom-fragment-types
> `showcontent`: Whether if the fragment contents should be included in the news file. If omitted, it defaults to true

So, you get just a list of the issue/pr/file numbers as at https://github.com/aio-libs/yarl/blob/68e7d9312ad9eccb695dbc174a4705e9a238c1fa/CHANGES.rst#misc-1.  Note that if you follow those, their content isn't empty, just ignored.  https://github.com/aio-libs/yarl/pull/647/files#diff-3c671c897bfbc0b283bbb224c8784fda9364a40d0233748d2606c6b78faa0394

Reply:
Fun, so GitHub let me edit the 'empty' file that actually had 'two lines' and delete one and commit it and it still seems to be the same file unchanged...  But, you've got a linter complaining about it so I'll go ahead and delete it since that seems to be process here.

Reply:
Not to be pounding on this...  but I got the doc link so may as well finalize the references here.

https://docs.twisted.org/en/latest/development/dev-process.html#release-notes-management
> **misc** - Issues which are very minor and not worth summarizing outside of the git changelog. These should be empty (their contents will be ignored)

And the PR fixing the doc link is an example of an empty misc fragment.

https://github.com/twisted/twisted/pull/11777/files

Reply:
FWIW I've been considering showing that content at some point. Too many people misuse these fragments and put really useful stuff there. And it's hard to catch for humans during reviews.

Reply:
It's all in your intent in the setup.  Nice and configurable.

To make fragments a bit more reviewable, at least in terms of viewing them rendered in case there's any markup errors, I added a step to dump the draft output to the GitHub workflow summary.

https://github.com/Chia-Network/chia-blockchain/pull/13882/files#diff-b1b82537432942b954fa8846c9e80709d0d919428ed1e1692d7c0bc4bc8271ecR112-R114
```yaml
    - name: Create single version changlog summary
      run: |
        towncrier build --yes --draft >> $GITHUB_STEP_SUMMARY
```
(not sure what the `--yes` is doing there...)

Sample output from this at https://github.com/Chia-Network/chia-blockchain/actions/runs/3445535929#summary-9436412167.

Reply:
@altendky yea, I actually often make the integration with both Sphinx (via by extension @ https://github.com/sphinx-contrib/sphinxcontrib-towncrier — can error out in nitpicky mode) and GH job summary (requires pandoc for RST->md). See:
- https://github.com/ansible/pylibssh/actions/runs/3533848038#summary-9670210175
- https://github.com/ansible/pylibssh/actions/runs/3533848038/workflow#L469-L516
- https://ansible-pylibssh.rtfd.io/changelog